README.md

```diff
@@ -1,2 +1,99 @@
-# Hello-world
-Hello Microverse Project
+<a name="readme-top"></a>
+<div align="center">
+
+  <img src="https://github.com/mohisa302/ReadMe/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+**[Hello Microverse]** is a begining in my Microverse journey!
+
+
+## 🛠 Built With <a name="built-with"></a>
+### Key Features <a name="key-features"></a>
+- **[using linters]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+new developers can use this project to get familiar with basic programmers tools and templates
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+@mohisa302
+
+👤 **Author1**
+
+- GitHub: [@mohisa302](https://github.com/mohisa302)
+- Twitter: [@Kholochelam](https://twitter.com/Kholochelam)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mohadese-sadeghi-692551199/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[will improve page elements]**
+- [ ] **[will improve styles]**
+- [ ] **[will add js]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my favorite mentor Houda Mzari
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+- **[How this project help me in my first days?]**
+
+  - [This project includes most of the tools which you are going to use for your codes]
+
+- **[What is Microverse?]**
+
+  - [Microverse is one of the most awesome coding school!]
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] There is one small change in your readme file. Kindly go through the following steps to add your LICENSE to your project as it is recommended to use the [MIT license](https://choosealicense.com/licenses/mit/)
* Please add a LICENSE.md file in your project root folder and add [this content](https://github.com/microverseinc/readme-template/blob/master/MIT.md) in your file and modify the content accordingly (for example, your name, app type) 😃
* Kindly, add a link to your README
> This project is [MIT](https://github.com/Fene-87/Hello-World/pull/LICENSE) licensed. 😃

Reply:
@RiyaBulia12 would mind giving me hand with linters? I added yml file to the feature-a branch and also check linters have passed checks 

![image](https://user-images.githubusercontent.com/96848068/204745584-76817377-d90a-4ad9-a33e-386fd99240a7.png)

Reply:
> @RiyaBulia12 would mind giving me hand with linters? I added yml file to the feature-a branch and also check linters have passed checks
> 
> ![image](https://user-images.githubusercontent.com/96848068/204745584-76817377-d90a-4ad9-a33e-386fd99240a7.png)

Hi @mohisa302,

When I looked at your linters checks it was showing me `0` instead of `4`. Not sure what happened. Apologies for the confusion. You are good to request another review if you have made changes related to ./LICENSE in your README file

Reply:
Hello @RiyaBulia12 ,
Thanks for your quick response 🙂, sure I made changes so I'm going to request a review. 
best wishes